ding/model/template/efficientzero/efficientzero_base_model.py

```diff
@@ -0,0 +1,153 @@
+"""
+The following code is adapted from https://github.com/YeWR/EfficientZero/blob/main/core/model.py
+"""
+
+import torch
+import typing
+
+import numpy as np
+import torch.nn as nn
+
+from typing import List
+
+
+class NetworkOutput(typing.NamedTuple):
```

Comment:
move to dataclass, you can refer to [this implementation](https://github.com/opendilab/DI-engine/blob/main/ding/data/buffer/buffer.py#L41)